libs/internal/src/events/context_key_cache.cpp

```diff
@@ -0,0 +1,32 @@
+#include <launchdarkly/events/lru_cache.hpp>
```

Comment:
Filename of this one is still context_key_cache. 
But maybe if this cache is only for strings, then it doesn't need to be generic.

(Big segments, if we do it at some point has three states. Basically a tribool. Included in the segment, not included in the segment, or it doesn't exist. So when it isn't part of the segment we still cache that.)

Reply:
Uhg, sorry for the churn. Renamed to `lru_cache` and renamed the test file as well.

Reply:
No biggie.